.travis.yml

```diff
@@ -17,20 +17,24 @@
 # limitations under the License.
 
 sudo: true
-dist: xenial
+dist: trusty
```

Comment:
why are we going back in time?

Reply:
because xenial is not officially supported by travis and builds were failing (I've switched it on master already)

Reply:
:+1: , for info: https://docs.travis-ci.com/user/reference/overview/